### DISCUSSION TITLE
introductory doc on basic usage with laggraph doesn't work

### DISCUSSION THREAD

**Discussion by jeswincyriac** [mem0ai/mem0#1897] [Q&A] [answered]
Followed https://docs.mem0.ai/integrations/langgraph

mem0 = Memory(api_key=MEM0_API_KEY) fauiled with Memory.__init__() got an unexpected keyword argument 'api_key'

**Comment by Dev-Khant** [comment, marked as answer]
Hey @jeswincyriac Thanks for sharing this issue. It has been fixed now. 
Please check: https://docs.mem0.ai/integrations/langgraph

**Reply by jeswincyriac** [reply]
Thanks!
